docs/conch/examples/sshsimpleserver.py

```diff
@@ -49,14 +49,6 @@
 $ ssh -p 5022 -i ssh-keys/client_rsa user@localhost
 """
 
-# List to public SSH keys accepted by the server.
-# One public key per line
-# You should add here the content of ssh-keys/client_rsa.pub
-CLIENT_PUBLIC = """
-sk-ssh-ed25519@openssh.com AAAAGnNrLXNzaC1lZDI1NTE5QG9wZW5zc2guY29tAAAAIKULA+QMG6KFFXeooTZ9dHXfV9wsUzvjBcCHvWJgEYKcAAAAE3NzaDplZDI1NTE5LXl1YmktNWM= user@test
-""".strip()
-
-
```

Comment:
Would we need to add a file to make the example work again? Or maybe tell the user how to generate the file.